### ISSUE TITLE
Map is not generating correctly

### ISSUE THREAD

**Issue by guptachetan1997** [SMILEY4/ktor-openapi-tools#20]
When we have a data class eg:
```kotlin
data class MyClass(val x: Map<String, String>)
```

This gets generated as:
```json
"x": {
"type": "object"
},
```

Additional properties should be added. Otherwise the client generated by this open api json treats the field as String and not a map.

Something like this:

```json
"x": {
"type": "object",
"additionalProperties": { "type": "string" }
},
```

**Comment by SMILEY4** [comment]
Hi,
thanks for reporting the issue. This is bug should now be fixed with the new version 1.0.2.

**Comment by blinchk** [comment]
Looks fixed with f288486f4c3823a4b6c13df90bff65a63c1159a4 and [1.0.2](https://github.com/SMILEY4/ktor-swagger-ui/releases/tag/1.0.2).

**Comment by guptachetan1997** [comment]
Yes this is working for me now. Thanks for the fix.
